src/utils/component-utils.tsx

```diff
@@ -0,0 +1,13 @@
+import { BaseProps } from "../models/BaseProps";
+
+/**
+ * Returns the props to be spreaded at component container level
+ * @param param0 component base props
+ */
+export function getContainerOps({ testId }: BaseProps) {
```

Comment:
I don't really get what does this function by reading at the name, and even in general I don't really understand its purpose, could you explain a bit more?

Reply:
I removed.
I guess we would need to some kind of base properties for the component containers, at least so we can target them at test. But with the lint forcing me to include role for example, we can manage it without for now.